feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64ff24b80431f62ec6b93f65.md

---
id: 64ff24b80431f62ec6b93f65
title: Step 52
challengeType: 0
dashedName: step-52
---

# --description--

> Remove the `myTaskArr` array and every other code with which you've learned local storage because you don't need them anymore.

Reviewer: ```suggestion
Remove the `myTaskArr` array and all of the code for `localStorage` because you don't need them anymore.
```
